### DISCUSSION TITLE
!!THEY DO NOT ACTIVATE MY GIT HUB STUDENT PACK

### DISCUSSION THREAD

**Discussion by G-YHONNER** [community/community#129111] [GitHub Education]
### Hi everyone,

Hello everyone, I have been waiting for 4 days for the benefits of Github for students to be activated, I have already been verified and so far they have not done it, is there something I can do.  I am from peru

**Comment by davevad93** [comment]
Hi @YHONNER-VB ,

Take a look at this discussion #129051 

If the status of your application says "`Your benefits will be available soon`", it means that you still have to wait a few days.

**Reply by G-YHONNER** [reply]
![IMG_20240619_162929.jpg](https://github.com/community/community/assets/157642525/0f23b5ed-3273-4ddb-9385-fd6d58462b4f)

**Reply by G-YHONNER** [reply]
How much longer do I have to wait approximately?

**Reply by davevad93** [reply]
Yes, you've been approved but you're benefits will be available soon, when available your application status will turn purple and change to:

`Congrats! Your request for @YHONNER-VB was approved and your benefits are now available`

Be patient because It may take a while, from a few days to a few weeks. 👍

**Reply by davevad93** [reply]
> How much longer do I have to wait approximately?

Usually it takes just a few days but the waiting time may vary.
